include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
re: installing config.h

It would be, because the CppUTestConfig.h would need to include it, so there *all* the #defines that are generated in the config.h will be in the same namespace.

E.g. if I do, in my test code:

#include <CppUTest/TestHarness.h>  <--- will include config.h
#include <myapp.h>

Not the behavior of myapp.h *might* be influence by the config.h #defines. If that would happen, it would be exceptionally painful to find. The solution is to rename all the #defines in the config.h (which autotools provides some support for), but it would require quite some additional autotools work.